src/utils/sentry.ts

```diff
@@ -31,6 +31,18 @@ const sentry =
               matchRoutes,
             ),
           }),
+          Sentry.httpClientIntegration({
+            failedRequestStatusCodes: [
+              [400, 499],
+              [500, 599],
```

Comment:
Wouldn't this be 400-599?

Reply:
Yes exactly - i supposed i could have used a single range specifier instead of 2